kibana/tests/kibana_test.py

```diff
@@ -427,6 +427,21 @@ def test_service_annotations():
     ]
     assert s == "0.0.0.0/0"
 
+def test_adding_a_loadBalancerIP():
+    config = ""
+
+    r = helm_template(config)
+
+    assert "loadBalancerIP" not in r["service"][uname]["spec"]
+
+    config = """
+    service:
+      loadBalancerIP: 12.4.19.82
+    """
+
+    r = helm_template(config)
+
+    assert r["service"][uname]["spec"]["loadBalancerIP"] == "12.4.19.82"
```

Comment:
Black formatter requires 2 blanck lines between each functions:
```suggestion
    assert r["service"][uname]["spec"]["loadBalancerIP"] == "12.4.19.82"
    
```